codebundles/aws-c7n-tag-compliance/.runwhen/templates/aws-c7n-tag-compliance-slx.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: runwhen.com/v1
+kind: ServiceLevelX
+metadata:
+  name: {{slx_name}}
+  labels:
+    {% include "common-labels.yaml" %}
+  annotations:
+    {% include "common-annotations.yaml" %}
+spec:
+  imageURL: http://IMAGE_URL_PLACEHOLDER
+  alias: Checks for missing tags on AWS resources in AWS account {{match_resource.resource.account_id}}
```

Comment:
This is a description, not an alias. The Alias would be something like "AWS Tag Compliance for Account ${ACCOUNT}'